components/ipfs/ipfs_utils.cc

```diff
@@ -61,38 +59,40 @@ bool IsValidCID(const std::string& cid) {
   return base::StartsWith(cid, kCIDv0Prefix);
 }
 
-bool IsIpfsDisabledByPolicy(content::BrowserContext* context) {
-  DCHECK(context);
-  PrefService* prefs = user_prefs::UserPrefs::Get(context);
-  return prefs->FindPreference(kIPFSEnabled) &&
-         prefs->IsManagedPreference(kIPFSEnabled) &&
-         !prefs->GetBoolean(kIPFSEnabled);
-}
-
-bool IsIpfsEnabled(content::BrowserContext* context) {
-  DCHECK(context);
-  if (context->IsOffTheRecord() || IsIpfsDisabledByPolicy(context) ||
```

Comment:
I removed IsOffTheRecord check because we have similar here https://github.com/brave/brave-core/blob/master/browser/ipfs/ipfs_service_factory.cc#L35, wdyt? @bbondy ?